src/warnet/admin.py

```diff
@@ -33,3 +34,132 @@ def init():
         f"[green]Copied network and namespace example files to {Path(current_dir) / NETWORK_DIR.name}[/green]"
     )
     richprint(f"[green]Created warnet project structure in {current_dir}[/green]")
+
+
+# Get kubectl values
+def get_kubectl_value(jsonpath):
+    return subprocess.check_output(
+        ["kubectl", "config", "view", "--minify", "-o", f"jsonpath={jsonpath}"]
+    ).decode("utf-8")
+
+
+# Get all namespaces that start with "warnet-"
+def get_warnet_namespaces():
```

Comment:
if we are going to start having conventions around namespace strings (e.g., warnet-), might be more useful to make this a "get_namespaces_by_prefix(prefix: str)" function. not urgent, though

Reply:
I agree with @josibake but more strongly. I think we should avoid using any naming conventions like we did before helm (if possible) -- can we use kubernetes metadata "labels" instead?